packages/babel-helpers/src/helpers/wrapRegExp.js

```diff
@@ -31,7 +31,13 @@ export default function _wrapRegExp() {
         this,
         str,
         substitution.replace(/\$<([^>]+)>/g, function (_, name) {
-          return "$" + groups[name];
+          var group = groups[name];
+          if (typeof group === "number") return "$" + group;
+          return group
+            .map(function (g) {
+              return "$" + g;
+            })
+            .join("");
```

Comment:
We can save some bytes by doing
```suggestion
          return "$" + (typeof group === "number" ? group : group.join("$"));
```